src/Raven.Server/Json/Converters/SliceJsonConverter.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Server.ServerWide.Context;
+using Sparrow;
+using Sparrow.Json;
+using Voron;
+
+namespace Raven.Server.Json.Converters
+{
+    internal sealed class SliceJsonConverter : RavenJsonConverter
+    {
+        public static readonly SliceJsonConverter Instance = new SliceJsonConverter();
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (!(writer is BlittableJsonWriter blittableJsonWriter))
+            {
+                throw new SerializationException($"{nameof(BlittableJsonWriter)} must to be used for write {nameof(LazyStringValue)}");
+            }
+
+            if (!(value is Slice slice))
+            {
+                throw new SerializationException($"Try to write non {nameof(Slice)} value with {nameof(SliceJsonConverter)}");
+            }
+
+            var buffer = new byte[slice.Size];
```

Comment:
Base we use `Base64` and avoid the managed allocations?

Reply:
Issue created
[RavenDB-11852](http://issues.hibernatingrhinos.com/issue/RavenDB-11852)